using-mobile-extensions/adobe-target/target-api-reference.md

```diff
@@ -1,591 +1,755 @@
-
-ACPTarget.locationClicked(withName: "cartLocation", targetParameters: targetParameters)
+ACPTarget.getThirdPartyId({thirdPartyID in
+       // read Target thirdPartyId
+})
 ```
-{% endtab %}
 
-{% tab title="React Native" %}
-#### Syntax
+**Objective-C**
 
-```javascript
-locationClickedWithName(name: string, parameters?: ACPTargetParameters)
+```objc
+[ACPTarget getThirdPartyId:^(NSString *thirdPartyId){
+       // read Target thirdPartyId
+}];
 ```
 
-* _name_ is a string that contains the mbox location for which the click notification will be sent to Target.
-* _parameters_ is the configured `ACPTargetParameters` for the request.
+{% endtab %}
 
-#### Example
+{% tab title="React Native" %}
 
-**JavaScript**
+**Syntax**
 
 ```javascript
-// Mbox parameters
-var mboxParameters = {"membership": "prime"};
-
-// Product parameters
-var productParameters = new ACPTargetProduct("CEDFJC", "Electronics");
-
-// Order parameters
-var orderParameters = new ACPTargetOrder("NJJICK", 650, ["81","123","190"]);
+getThirdPartyId(): Promise<string>
+```
 
-// Profile parameters
-var profileParameters = {"ageGroup": "20-32"};
+- A _Promise_ object is returned and is resolved with the `thirdPartyId` value.
 
-// Create Target parameters
-var product = new ACPTargetProduct("24D334", "Stationary");
-var order = new ACPTargetOrder("ADCKKBC", 400.50, ["34","125"]);
-var targetParameters = new ACPTargetParameters(null, null, product, order);
+**Example**
 
-ACPTarget.locationClickedWithName("cartLocation", targetParameters);
+```javascript
+ACPTarget.getThirdPartyId().then(thirdPartyId => {
+            // read Target thirdPartyId 
+});
 ```
+
 {% endtab %}
 {% endtabs %}
 
-## locationsDisplayed
-
-Use this API to send a location \(mbox\) display notification to the configured Target server. This API should be invoked for a prefetched mbox after the mbox content is retrieved using the `retrieveLocationContent` API. If no previous prefetch request is made, and the mbox content is retrieved using the `retrieveLocationContent` API, calling this API does not trigger a notification request to the Target server.
-
-{% hint style="warning" %}
-Do not use this API with the deprecated [loadRequests](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference-deprecated#load-target-requests) API. For a prefetched mbox, after calling the deprecated `loadRequests` API, the mbox display notification is sent internally to the Target server by the SDK.
-{% endhint %}
+---
+### getTntId
+This API gets the Target user ID (also known as the `tntId`) from the Target service. The `tntId` is returned in the network response after a successful call to `prefetchContent`, which is then persisted in the SDK. This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall or when the `resetExperience` API is used.
```

Comment:
The tntId is returned in the response for both `prefetchContent` and `retrieveLocationContent`. Can we fix the description please?